### PR TITLE
chore: Update CODEOWNERS

### PR DESCRIPTION
**Pull request by sai-sunder-s**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/google-auth-library-python-httplib2/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Comment by sai-sunder-s** [comment]
Before merging, the group needs to be added manually to the repo as well I think

**Comment by dandhlee** [comment]
@parthea, could you give write access? Confirmed that the team exists: https://github.com/orgs/googleapis/teams/googleapis-auth

**Review by parthea — APPROVED** [review]

**Pull request merged**
